### PR TITLE
Rename Permission to CommandPermission and Add custom attributes to SlashCommand

### PR DESCRIPTION
**Pull request by TheReaper62**
## Summary

Rename Permission to CommandPermission fixes [Issue#416](https://github.com/Pycord-Development/pycord/issues/416)
Add custom attributes to SlashCommand fixes [Issue#426](https://github.com/Pycord-Development/pycord/issues/426)

## Checklist

- [x] This PR fixes 2 issues.

### CODE REVIEW

**Comment by RPSMain** [comment]
by the way "as per" isn't a [issue keyword](https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests)

File: discord/commands/commands.py [hunk 1]
```diff
@@ -372,12 +372,19 @@ def __init__(self, func: Callable, *args, **kwargs) -> None:
         self.callback = func
 
         self.guild_ids: Optional[List[int]] = kwargs.get("guild_ids", None)
+        kwargs.pop("guild_ids", None)
```
Comment (BobDotCom) [inline discord/commands/commands.py:375]:
```suggestion
        self.guild_ids: Optional[List[int]] = kwargs.pop("guild_ids", None)
```

**Review by BobDotCom — CHANGES_REQUESTED** [review]

**Review by Lulalaby — APPROVED** [review]

**Comment by TheReaper62** [comment]
Stale PR

**Comment by Lulalaby** [comment]
Closing as stale. Follow-up: #662

**Pull request closed without merging**